pages/page.jsx

```diff
@@ -1,8 +1,8 @@
-import { Card, Page, Layout, TextContainer, Heading } from '@shopify/polaris'
+import { Card, Page as PolarisPage, Layout, TextContainer, Heading } from '@shopify/polaris'
 
-export default function Page2() {
+export default function Page() {
   return (
-    <Page>
+    <PolarisPage>
```

Comment:
What is special about this PolarisPage? I don't think we use it in the QR Code app. Should we be using it there? Should we be using it for the index page in the app template?